Make mirengue procedure

dmbrasw2 opened the pull request:
Wrote a rough draft of the procedure sections and steps.

Included a couple of images for clarity

lingeringcode commented:
Overall feedback comments here.

## Staging Comments

- [Feedback about Ingredients](https://github.com/ENG517/Procedure/blob/0a90b409227eb811f85a8ef4ff162553f9fdae52/lindgrenc/p-exzprezzo5000-make-espresso.md?plain=1#L13):
  - Describe the problem I see in this section. Evaluate the content based on our shared criteria about staging content. Suggest a possible revision.
  - Describe the problem I see in this section. Evaluate the content based on our shared criteria about staging content. Suggest a possible revision.

lingeringcode commented:
Here's changed based on your generous feedback:

- Change 1 based on X and Y's suggestions about Z staging comments.
- etc
- etc

dmbrasw2 committed:
Initial setup

Copied and renamed TOCOPY

dmbrasw2 committed:
Made branch 2

Renamed task 2 to make meringue and made a make meringue branch

dmbrasw2 committed:
Adding images

Added images to image repository, will put in file later

dmbrasw2 committed:
Fixed image filename

dmbrasw2 committed:
Meringue rough draft

Wrote a rough draft of the meringue procedure. Included a few images.
Added images to the meringue repository

dmbrasw2 committed:
Added a Pull Request file

Added the pull request file for Meringue to my repository
